src/spec.c

```diff
@@ -262,21 +275,23 @@ static void IndexSpec_TimedOutProc(RedisModuleCtx *ctx, WeakRef w_ref) {
     // the spec was already deleted, nothing to do here
     return;
   }
-  RedisModule_Log(RSDummyContext, REDISMODULE_LOGLEVEL_VERBOSE, "Freeing index %s by timer", sp->name);
+  char name[MAX_OBFUSCATED_INDEX_NAME];
+  Obfuscate_Index(sp->uniqueId, name);
```

Comment:
Function/macro..

Reply:
I don't understand, do you mean you want me to change this to a macro?

Reply:
Like previous comments.. Yes to save from needing to change this everywhere later.